meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-development.bb

```diff
@@ -31,7 +31,7 @@ PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " optee-test"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " openssh-sftp-server"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " ${@bb.utils.contains('COMBINED_FEATURES', 'usbgadget', 'systemd-net-conf-dbg-iface', '', d)}"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " systemd-analyze"
-
+PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " systemd-watchdog-conf"
```

Comment:
```suggestion
```
This is not needed since in `meta-mbl-distro/classes/mbl-distro-image-development.bbclass` we include the packagegroup-mbl-production.